internal/tracker/ci_test.go

```diff
@@ -70,5 +74,56 @@ func TestCITracker(t *testing.T) {
 			c.FailedComputeSimilarityID()
 			require.Equal(t, 1, c.FailedSimilarityID)
 		})
+		t.Run(fmt.Sprintf(tt.name+"_GetOutputLines"), func(t *testing.T) {
+			got := c.GetOutputLines()
+			if !reflect.DeepEqual(got, 3) {
+				t.Errorf("GetOutputLines() = %v, want = %v", got, 3)
+			}
+		})
+	}
+}
+
+// TestNewTracker tests the functions [NewTracker()] and all the methods called by them
+func TestNewTracker(t *testing.T) {
+	type args struct {
+		outputLines int
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    CITracker
+		wantErr bool
+	}{
+		{
+			name: "test_new_ci_tracker",
+			args: args{
+				outputLines: 3,
+			},
+			want: CITracker{
+				lines: 3,
+			},
+			wantErr: false,
+		},
+		{
+			name: "test_tracker_error",
+			args: args{
+				outputLines: 0,
+			},
+			want:    CITracker{},
+			wantErr: true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := NewTracker(tt.args.outputLines)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("NewTracker() error = %v, wantErr = %v", err, tt.wantErr)
```

Comment:
use `test.StringifyStruct` to make the output easier to inspect in case of failure

Reply:
resolved